components/infobox/wikis/rainbowsix/infobox_league_custom.lua

```diff
@@ -0,0 +1,306 @@
+	local args = _league.args
+	table.insert(widgets, Cell{
+		name = 'Teams',
+		content = {(args.team_number or '') .. (args.team_slots and ('/' .. args.team_slots) or '')}
+	})
+	table.insert(widgets, Cell{
+		name = 'Game',
+		content = {CustomLeague:_createGameCell(args)}
+	})
+	table.insert(widgets, Cell{
+		name = 'Players',
+		content = {args.player_number}
+	})
+
+	return widgets
+end
+
+function CustomInjector:parse(id, widgets)
+	local args = _league.args
+	if id == 'customcontent' then
+		if not String.isEmpty(args.map1) then
+			local game = not String.isEmpty(args.game) and ('/' .. args.game) or ''
+			local maps = {CustomLeague:_makeInternalLink(args.map1 .. game .. '|' .. args.map1)}
+			local index = 2
+
+			while not String.isEmpty(args['map' .. index]) do
+				local map = args['map' .. index]
+				table.insert(maps, '&nbsp;• ' ..
+					tostring(CustomLeague:_createNoWrappingSpan(
+						CustomLeague:_makeInternalLink(map .. game .. '|' .. map)
+					))
+				)
+				index = index + 1
+			end
+			table.insert(widgets, Title{name = 'Maps'})
+			table.insert(widgets, Center{content = maps})
+		end
+
+
+		if not String.isEmpty(args.team1) then
+			local teams = {CustomLeague:_makeInternalLink(args.team1)}
+			local index = 2
+
+			while not String.isEmpty(args['team' .. index]) do
+				table.insert(teams, '&nbsp;• ' ..
+					tostring(CustomLeague:_createNoWrappingSpan(
+						CustomLeague:_makeInternalLink(args['team' .. index])
+					))
+				)
+				index = index + 1
+			end
+			table.insert(widgets, Center{content = teams})
+		end
+	elseif id == 'prizepool' then
+		return {
+			Cell{
+				name = 'Prize pool',
+				content = {CustomLeague:_createPrizepool()}
+			},
+		}
+	elseif id == 'liquipediatier' then
+	    local ubisoftTier = _args.ubisofttier
+		if not String.isEmpty(ubisoftTier) then
+			widgets = {
+				Cell{
+					name = 'Ubisoft tier',
+					content = {'[[Ubisoft|' .. ubisoftTier .. ']]'},
+					classes = {'valvepremier-highlighted'}
+				}
+			}
+		end
+		table.insert(widgets, Cell{
+			name = 'Liquipedia tier',
+			content = {CustomLeague:_createLiquipediaTierDisplay()},
+			classes = {String.isEmpty(_args['ubisoft-sponsored']) and '' or 'valvepremier-highlighted'}
+		})
+		table.insert(widgets, Cell{
+			name = 'Ubisoft tier',
+			content = {Tier['ubisoft'][string.lower(_args.eatier or '')]},
+			classes = {'valvepremier-highlighted'}
+		})
+		return widgets
+	end
+	return widgets
+end
+
+function League:addToLpdb(lpdbData, args)
+	lpdbData.maps = CustomLeague:_concatArgs('map')
+
+	lpdbData.game = _game or args.game
+	lpdbData.publishertier = args.ubisofttier
+	lpdbData.participantsnumber = args.player_number or args.team_number
+	lpdbData.extradata = {
+		prizepoollocal = Variables.varDefault('prizepoollocal', ''),
+		startdate_raw = Variables.varDefault('raw_sdate', ''),
+		enddate_raw = Variables.varDefault('raw_edate', ''),
+        individual = not String.isEmpty(args.player_number),
```

Comment:
current method of determine if it's an individual is not checking `if args.individual == 'true'`